build_overrides/ot_efr32.gni

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+declare_args() {
+  openthread_efr32_root = "//third_party/openthread/ot-efr32"
```

Comment:
Wondering if we should but all those path definition in the existing openthread.gni

declare_args() {
  openthread_root = "//third_party/openthread/repo"
  openthread_efr32_root = "//third_party/openthread/ot-efr32"
  openthread_nxp_root = "//third_party/openthread/ot-nxp"
  openthread_qorvo_root = "//third_party/openthread/ot-qorvo"
}

Reply:
That would indeed avoid the extra override files.
A step further, as the use of these args are isolated to the `/third_party/openthread/platforms/<platform>/BUILD.gn` files, maybe it makes even more sense to not define a variable at all and just declare it locally there.
You only need the `openthread_<platform>_root` var for the OT platform glue code
Not sure however I completely follow the GN design reasoning with this suggestion.

Reply:
The proposed change is now integrated to this PR. Thanks!